_docs/Salesforce-Integration.md

```diff
@@ -235,4 +235,140 @@ To integrate Salesforce with Visual Studio Code, see [Salesforce Extension Pack]
 Once installed and you've authenticated to the Salesforce Sandbox, you can use the context menu to manage deployment and retrieval of your Salesforce components.
 
 ![The context menu within Visual Studio Code](/assets/images/salesforce-integration/vsc-deploy.png){:class="img-responsive"}
-</div>
\ No newline at end of file
+</div>
+
+### Implementing custom localization
+
+In order to implement localization for translations that aren't out of the box in your Coveo for Salesforce components, we will leverage the component's static resources.
+
+In order to simplify development, you can and should use Turbo's CLI commands regarding localization. Let us assume we created the following translation files using the CLI commands as an example:
+
+```
+// en.js
+{
+    "All": "All",
+    "Articles": "Articles"
+}
+```
+
+```
+// fr.js
+{
+    "All": "Tout",
+    "Articles": "Articles"
+}
+```
+
+Note that every key matches a `data-caption`.
```

Comment:
data-caption or the value captured by the [`Coveo.l`](https://docs.coveo.com/en/421/javascript-search-framework/change-the-language-of-your-search-interface#getting-the-localized-value-of-a-single-string) function

Reply:
Updated.